tools/protoxform/BUILD

```diff
@@ -8,11 +8,32 @@ py_binary(
         "protoxform.py",
         "utils.py",
     ],
-    data = ["//:.clang-format"],
     python_version = "PY3",
```

Comment:
nit: I think any instances of `python_version = "PY3",` can be removed now? I guess I should have done this in my other PR.